charts/kubearchive/values.yaml

```diff
@@ -80,8 +86,8 @@ database:
   # database basic configuration
   postgresData:
     # database name
-    dbName: ""
+    dbName: "postgresdb"
     # database user
-    dbUser: ""
+    dbUser: "ps_user"
     # database password
-    dbPassword: ""
+    dbPassword: "P0stgr3sdbP@ssword"  # notsecret
```

Comment:
add a comment here to mark this vale as not a secret
`dbPassword: "P0stgr3sdbP@ssword"` # notsecret

Reply:
nvm, you already added it :)

Reply:
The email from RedHat security was faster :rofl: